html-templates/sbg/progress/section-term-reports/sbg-print.tpl

```diff
@@ -1,52 +0,0 @@
-{extends "progress/section-term-reports/sectionTermReports.tpl"}
```

Comment:
`php-config/Slate/Progress/SectionTermReportsRequestHandler.config.d/print-template.php` is still pointing at this file